doc/src/usage/coverage.md

```diff
@@ -13,6 +13,10 @@ Both a host directory and a corpus directory must be mounted.
 Example:
 
 ```
+export HOST_OUTPUT_DIR="$(pwd)/coverage-output"
+export HOST_CORPUS_DIR="$(pwd)/your-corpus"
+export SCENARIO="scenario-name"
```

Comment:
Maybe just:

```suggestion
export SCENARIO="name"
```

Otherwise one might put `scenario-ir` which then resolves to `scenario-scenario-ir` below.